fuuk/people/migrations/0003_refactor_grants.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('people', '0002_markdown'),
```

Comment:
Django don't let me change `0002_markdown` to `0002_manytomany_null` and wants me to merge the two migration branches.